### ISSUE TITLE
RealBlur data for training

### ISSUE THREAD

**Issue by asdf1996** [swz30/MPRNet#40]
Hi,
I'd like to know if you have pre-processed the data of RealBlur dataset[http://cg.postech.ac.kr/research/realblur/](url).
Since I directly use those data for training, the loss becomes inf.
The learning rate and batch size are as default settings.  
I'd like to know how you select the data for training.Thanks!

**Comment by adityac8** [comment]
Hi @asdf1996 

In Table 4 of MPRNet, we report two results for RealBlur datasets.
![realblur](https://user-images.githubusercontent.com/19192257/116200041-e8ca7b80-a748-11eb-94d4-83084bbce6e4.png)
For the top part, we simply take the pretrained model of GoPro dataset and test it on RealBlur datasets.
For the bottom part, we train two separate models, each for RealBlurR and RealBlurJ. For this, we follow the same training strategy as the authors of RealBlur dataset. We use `GoPro+BSD+RealBlurR` for training our `RealBlurR` model and `GoPro+BSD+RealBlurJ` for training our `RealBlurJ` model. We train the models for 100 epochs.

Thanks

**Comment by asdf1996** [comment]
Thanks for reply.
So is it better to train the model from scratch， instead of using pre-trained model of GoPro and fine-tuning on RealBlur dataset?

**Comment by BZboys** [comment]
I had the same problem. The loss becomes inf in the task of deblur, and reducing the learning rate is useless.

**Comment by swz30** [comment]
> Thanks for reply.
> So is it better to train the model from scratch， instead of using pre-trained model of GoPro and fine-tuning on RealBlur dataset?

We did not try finetuning.

**Comment by asdf1996** [comment]
> We did not try finetuning.


Could you please also provide the model weights of trained on BSD and RealBlur dataset? THANKS!

**Comment by swz30** [comment]
Hi @asdf1996 

We use a slightly lighter network for the `GoPro+BSD+RealBlur` dataset. Please change n_feat=80
https://github.com/swz30/MPRNet/blob/435f483b6433a6cf8d43f4df9c20eebba47587bb/Deblurring/MPRNet.py#L239

You can download the weights from https://drive.google.com/drive/folders/1NY0nVT7A1w6cWTL8oig9e-ufaMT974mE?usp=sharing

**Comment by asdf1996** [comment]
Thank you!
